docs/java-helloworld.md

```diff
@@ -0,0 +1,145 @@
+---
+id: java-helloworld
+title: Java SDK "Hello World!" tutorial
+sidebar_label: Java "Hello World!"
+tags: helloworld, java, sdk, intellij, gradle, tutorial
+---
+
+This tutorial is aimed at developers who are new to [Temporal](/docs/overview), beginner Java programmers, or both. It will teach you how to build a simple "Hellow World!" Temporal Workflow application using the [Temporal Java SDK](https://github.com/temporalio/java-sdk) as well as a few basics, such as:
+
+- How to set up your development environment and its dependencies.
+- How to define an [Activity](/docs/activities) and a [Workflow](/docs/workflows).
+- How to create and execute a Worker program and a Workflow starter program.
+- Where to view Workflow details.
+
+## Prerequisites
+
+Make sure you have the tutorial [prerequisites](/docs/java-sdk-tutorial-prerequisites) setup.
+
+## Project setup
+
+Open IntelliJ and create a new Gradle project by following Step 1 of the [Getting started with Gradle guide](https://www.jetbrains.com/help/idea/getting-started-with-gradle.html#create_project). Make sure your project is created inside the "temporal-java-tutorials" directory and name your project "helloworld-tutorial". It will take a few moments to complete.
+
+Once Gradle has finished scaffolding you will need to customize the project dependencies. To do this, open the build.gradle file that is in the root of your project. Add the following lines to the dependencies section while replacing {version} with the most recent version listed on the [Temporal SDK Maven repository page](https://search.maven.org/artifact/io.temporal/temporal-sdk):
+
+```
+implementation 'io.temporal:temporal-sdk:{version}'
+implementation group: 'ch.qos.logback', name: 'logback-classic', version: '1.2.3'
+```
+
+The 'logback-classic' dependency will ensure that there is a logger to bind to within the SDK and prevent a default logger warning message.
+
+A "refresh" icon will appear on the screen, click it to load the changes. Gradle will rebuild with the dependencies.
+
+Lastly, to limit the logging output from the SDK, within src/main/resources create a logback.xml file and paste in the following XML:
```

Comment:
I appreciate you are trying to explain and not just command people, but this might be a bit too verbose. These things aren't practically important outside of doing it once.

Reply:
Having this makes sense in the context of splitting up tutorials I think.